frontend/src/index.jade

```diff
@@ -90,7 +90,8 @@ html
             .summary-chart(v-for="(user, i) in repo")
               .summary-chart__title(
                 title="click to view author's code",
-                v-on:click="$emit('view-authorship', {author:user.name, repo:user.repoName, name:user.displayName})"
+                v-on:click="$emit('view-authorship', " +
+                "{author:user.name, repo:user.repoName, name:user.displayName, totalCommits:user.totalCommits})"
```

Comment:
`+` operator should be on this line, also line wrap should include double the indentation from previous line.